src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -764,7 +764,7 @@ namespace HeatBalFiniteDiffManager {
 					mAlpha = 0.0;
 
 					//check for Material layers that are too thin and highly conductivity (not appropriate for surface models)
-					if ( Alpha > HighDiffusivityThreshold && ! Material( CurrentLayer ).WarnedForHighDiffusivity ) {
```

Comment:
It looks like the CTF's still use it. See here: https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/ConductionTransferFunctionCalc.cc#L369